controllers/auth-controllers.js

```diff
@@ -0,0 +1,82 @@
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+
+const User = require('../models/user');
+const { ctrlWrapper } = require('../utils');
+const { HttpError } = require('../helpers');
+
+const { SECRET_KEY } = process.env;
+
+const signUp = async (req, res) => {
+    const { email, password } = req.body;
+    const user = await User.findOne({ email });
+    if (user) throw HttpError(409, "Email in use");
```

Comment:
```suggestion
    if (user) {
     throw HttpError(409, "Email in use");
   }
```
Бажано використовувати фігурні дужки для кращої читабельності